backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py

```diff
@@ -0,0 +1,224 @@
+from abc import abstractmethod
+import ruamel
+from ruamel.yaml.scalarstring import SingleQuotedScalarString as single_quote
+from ruamel.yaml.scalarstring import DoubleQuotedScalarString as double_quote
+from SC4SNMP_UI_backend.common.backend_ui_conversions import get_group_or_profile_name_from_backend
+from ruamel.yaml.scalarstring import LiteralScalarString as literal_string
+import os
+from flask import current_app
+
+
+def bool_to_str(value):
+    if value:
+        return "t"
+    else:
+        return "f"
+
+
+class MongoToYamlDictConversion:
+    """
+    MongoToYamlDictConversion is an abstract class. Implementations of this class converts
+    appropriate mongo collections to dictionaries in such a way, that configurations from those collections can be
+    dumped to yaml file with appropriate formatting.
+    """
+    @classmethod
+    def yaml_escape_list(cls, *l):
```

Comment:
can you add comment with description of what this method do generally?

Reply:
Done